bbb-graphql-actions/src/actions/pluginDataAnalyticsSendObject.ts

```diff
@@ -0,0 +1,31 @@
+import { RedisMessage } from '../types';
+import {throwErrorIfInvalidInput} from "../imports/validation";
+
+export default function buildRedisMessage(sessionVariables: Record<string, unknown>, input: Record<string, unknown>): RedisMessage {
+  throwErrorIfInvalidInput(input,
+    [
+      {name: 'pluginName', type: 'string', required: true},
+      {name: 'dataAnalyticsObject', type: 'json', required: true},
```

Comment:
You could specify the expected value for children.
Take a look at `guestUsersSubmitApprovalStatus.ts` as example.

This way it will reject objects that doesn't fit the Dashboard necessary values.